frontend/taipy-gui/src/components/Taipy/Navigate.tsx

```diff
@@ -38,6 +38,9 @@ const Navigate = ({ to, params, tab, force }: NavigateProps) => {
                     navigate(0);
                 } else {
                     navigate({ pathname: to, search: `?${searchParams.toString()}` });
```

Comment:
do we want *2* navigate calls ?

Reply:
without the empty navigate call, the app would not do a hard reload. Please let me know if you have any other approach